src/components/expanded-state/asset/ChartExpandedState.js

```diff
@@ -135,10 +135,10 @@ const Spacer = styled.View({
 
 // truncate after the first paragraph or 4th dot
 function truncate(text) {
-  const firstParagraph = text?.split('\n')[0];
-  const first4Sentences = text?.split('.').slice(0, 4).join('.') + '.';
+  const firstParagraph = text.split('\n')[0];
```

Comment:
not those optional chaining do not make any sense

Reply:
Good point. Let's skip it then.